robotframework_metrics/robotmetrics.py

```diff
@@ -1168,17 +1174,17 @@ def generate_report(opts):
     rfmetrics.add_header('Content-Disposition',attachmentName)
     msg.attach(rfmetrics)
 
-    if send_email in ['True', 'true']:
```

Comment:
Similar change need to be done at line no.106 `if send_email in ['True', 'true']:` to `if send_email` as per latest changes. I will fix that!